relay-server/src/envelope.rs

```diff
@@ -866,6 +876,16 @@ impl Item {
         self.headers.spans_extracted = spans_extracted;
     }
 
+    /// Returns the ourlogs extracted flag.
+    pub fn ourlogs_extracted(&self) -> bool {
+        self.headers.ourlogs_extracted
+    }
```

Comment:
We should only extract in processing, then this becomes obsolete. This is just additional network traffic and compute to do it in PoPs.